BuildRPMDockerfile

```diff
@@ -0,0 +1,16 @@
+# Go binary builder
+FROM promscale_builder AS builder
```

Comment:
`build_rpm.sh` creates the `promscale_builder` image from `Dockerfile` the same file used to build the release image

Reply:
let's add this as code comment in the code